projects/archiver/src/services/editions-mappings.ts

```diff
@@ -0,0 +1,37 @@
+const createEditionToNameMap = () => {
+    const map = new Map<string, string>()
+    map.set('daily-edition', 'Daily Edition')
+    map.set('american-edition', 'American Edition')
+    map.set('australian-edition', 'Australian Edition')
+    map.set('training-edition', 'Training Edition')
+    return map
+}
+
+const createEditionToCountryMap = () => {
+    const map = new Map<string, string>()
+    map.set('daily-edition', 'uk')
```

Comment:
What's the mapping between country and edition for?

Reply:
for device notification,
But yeah it looks confusing, i agree
For now we will send the notifications only for Daily Editions, so i can keep the hardcoded option with comment

Reply:
i took that out :)